BinaryEncoding.md

```diff
@@ -356,7 +355,7 @@ a `data_segment` is:
 User-defined section string: `"name"`
 
 The names section does not change execution semantics, and thus is not allocated
-a section opcode.
+a section code.
 Like all user-defined sections, a validation error in this section does not cause validation of the module to fail.
```

Comment:
If making a distinct class of 'user-defined' sections then this wording is no long appropriate:

'like all user-defined sections' -> 'like all unknown sections'